src/components/ui/StatusBadge.svelte

```diff
@@ -1,60 +1,22 @@
 <script lang="ts">
-  import { createEventDispatcher } from 'svelte';
   import { getColorForStatus, Status, statusColors } from '../../utilities/status';
 
   export let status: Status = Status.Clean;
-  export let title: string = '';
-  export let titleWidth: string = '90px';
-
-  const dispatch = createEventDispatcher();
 
   let color: string = statusColors.blue;
 
   $: color = getColorForStatus(status);
 </script>
 
-<span class="status-badge" style="background: {color}" on:click={() => dispatch('click')}>
-  <span class="status">
-    {status}
-  </span>
-  <span class="title" style:width={titleWidth}>
-    <i class="bi bi-play-btn" />
-    {title}
-  </span>
+<span class="status" style="color: {color}">
```

Comment:
Can we just keep this class name as `.status-badge` for this commit? It should make the e2e tests pass. We can revisit with the final icon status design.

Reply:
Ah! yep I'll change that class name. The E2E tests were failing for a separate reason locally, which is what I was initially referring to, but makes sense this would break it too.